### PR TITLE
Implementing write service

### PR DESCRIPTION
**Pull request by jjrush**
#  #

## 🗣 Description ##

Implemented parsing for the WriteResponse and WriteRequest service. Added btest coverage for new logs that are generated.

## 💭 Motivation and context ##

Increase parsing coverage of opcua-binary protocol.

## 🧪 Testing ##

Ran the btest application and passed all 6 tests. Subsequently installed the parser with zkg and installed cleanly.

## ✅ Pre-approval checklist ##

- [ ] This PR has an informative and human-readable title.
- [ ] Changes are limited to a single goal - *eschew scope creep!*
- [ ] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: src/services/opcua_binary-services.pac [hunk 1]
```diff
@@ -14,128 +14,132 @@ type Service(msg_body: Msg_Body) = record {
     namespace_idx : uint8;
```
Comment (jjrush) [inline src/services/opcua_binary-services.pac:14]: Most of the changes in this file are merely cleanup. I moved all of the stubbed out/unimplemented services to the bottom below a comment and then alphabetized both the implemented and stubbed services within their respective regions.

**Comment by kkvarfordt** [comment]
Once merged, this PR will close Issue #45

**Comment by piercema** [comment]
The README needs to be updated with the information about the new log file. For example:
```Create Subscription Service (opcua-binary-create-subscription.log)
This log captures the details associated with calls to the Create Subscription Service.

See [create_subscription-types.zeek](https://github.com/cisagov/icsnpp-opcua-binary/blob/main/scripts/create-subscription-types.zeek) for a list of the fields logged.
See [opcua_binary-create_subscription.pac](https://github.com/cisagov/icsnpp-opcua-binary/blob/main/src/services/create-subscription/opcua_binary-create_subscription.pac), [opcua_binary-create_subscription_analyzer.pac](https://github.com/cisagov/icsnpp-opcua-binary/blob/main/src/services/create-subscription/opcua_binary-create_subscription_analyzer.pac), and [opcua_binary-create_subscription_debug.pac](https://github.com/cisagov/icsnpp-opcua-binary/blob/main/src/services/create-subscription/opcua_binary-create_subscription_debug.pac) for details on parsing, processing, and logging this service.```

**Comment by piercema** [comment]
The parser installs cleanly and all btests pass. A manual check shows log files are accurate.

**Review by piercema — APPROVED** [review]

**Pull request merged**